tracer/src/Datadog.Trace/Configuration/IntegrationSettings.cs

```diff
@@ -44,21 +44,21 @@ public IntegrationSettings(string integrationName, IConfigurationSource source)
         public string IntegrationName { get; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether
+        /// Gets a value indicating whether
         /// this integration is enabled.
         /// </summary>
-        public bool? Enabled { get; set; }
+        public bool? Enabled { get; internal set; }
```

Comment:
Why are we removing these settings? Users _can_ disable integrations using this property (see [docs](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-framework/?tab=code#automatic-instrumentation-integration-configuration)), and can enable analytics or set analytics sampling rate below (although the latter are deprecated).